template projects/telegram bot/dc2db3f2-46a2-431e-a901-386118a5a17b.json

```diff
@@ -0,0 +1,219 @@
+{
+  "attributes": {
+    "root": true,
+    "template": true
+  },
+  "description": "Basic telegram webhook with registration.\n\nThe webhook send back the message.\n\nPrecondition is a bot token.\nMore: https://core.telegram.org/bots/tutorial#obtain-your-bot-token",
+  "interface": {
+    "inputs": {},
+    "outputs": {}
+  },
+  "implementations": {
+    "cranq": {
+      "nodes": {
+        "8253ddd3-0709-42eb-b91e-309f5d3acf43": {
+          "coordinates": [
+            -691,
+            -8
+          ],
+          "description": "",
+          "params": {},
+          "prototype": "d227e22b-e827-4c9d-99db-f96da179bd52",
+          "name": "start"
+        },
+        "79688e50-8550-40b8-afea-3c7497fcb9f3": {
+          "coordinates": [
+            532,
+            287
+          ],
+          "description": "",
+          "params": {},
+          "prototype": "1d2a6438-e4bd-4105-b491-cebbb4cae348",
+          "name": "log"
+        },
+        "f3c274fb-6e25-4fde-93c2-2ee378a26c36": {
+          "coordinates": [
+            -392.5,
+            57.5
+          ],
+          "name": "bot params - FILL IT!!!",
+          "prototype": "321d0ab7-fd53-436d-8c08-8eff32103e5f",
+          "params": {
+            "cc0b6ce0-74f3-4f00-982b-d6dba32d5466": "{\"token\": \"\"}"
+          },
+          "description": "Set the token which was given by the BotFather.\n\nEg:\n```\n{\"token\": \"TOKEN\"}\n```\n\nHow to obtain one: https://core.telegram.org/bots/tutorial#obtain-your-bot-token\n\nHandle the verify token as a secret. Recommended solution is storing in evironment variable.\n\nMore: https://docs.cranq.io/toolbox/using-environment-variables"
+        },
+        "67b8d39e-a115-4d64-b705-c5ad1ac67ab6": {
+          "coordinates": [
+            -181.22500000000002,
+            248.375
+          ],
+          "name": "register webhook -  FILL IT!!!",
+          "prototype": "abdf3344-9a54-46b2-959a-0efdf9e193e1",
+          "params": {
+            "3822be7d-2186-45b8-a4f2-8a4fc15f9dad": "{\n  \"url\": \"WEBHOOK URL\"\n}"
```

Comment:
I would let the parameter unset

Reply:
This params is left here intentionally while others has been deleted and logged if missing?